gemini/use-cases/retrieval-augmented-generation/rag_vector_embedding_in_bigquery.ipynb

```diff
@@ -0,0 +1,794 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    sql = f"""CREATE OR REPLACE MODEL
</pre><p>nit: I think there's a way to directly run bigquery sql queries in jupyter notebooks, could be easier to read (with syntax-highlighting)</p><p>I recommend taking a look into it, but don't worry about it if it'll be tedious.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>